src/main/java/frc/bionic/swerve/MotorCANSparkMaxNeo.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Team 4909, Bionics
+ * Billerica Memorial High School
+ *
+ * Copyright:
+ *   2021 Bionics
+ *
+ * License:
+ *   MIT: https://opensource.org/licenses/MIT
+ *   See the LICENSE file in the project's top-level directory for details.
+ */
+
+package frc.bionic.swerve;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.MedianFilter;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+
+public class MotorCANSparkMaxNeo implements IMotor
+{
+  // Initial, default PID constants, overridden by persistent shuffleboard fields
+  private static final double kMotorP  = 0.000090;
+  private static final double kMotorI  = 0.000001;
+  private static final double kMotorD  = 0.000090;
```

Comment:
Wasn't D 100

Reply:
These were copied right from the old code. No, only IZone was 100.